src/ring_theory/integral_closure.lean

```diff
@@ -125,6 +125,19 @@ begin
     aeval_alg_hom_apply, aeval_map_algebra_map, aeval_def, hP.2, _root_.map_zero]
 end
 
+lemma is_integral_of_commuting_diagram {R S T U : Type*} [nontrivial S] [nontrivial T]
```

Comment:
Same comment as above about the name. This can maybe be golfed introducing some `is_scalar_tower`, but I am not sure.

Reply:
I gues using `is_scalar_tower` is equivalent to this, since we have `ring_hom.to_algebra`. Which one would you think is most useful/practical to use?